pilot/pkg/status/state.go

```diff
@@ -104,6 +105,11 @@ func (c *DistributionController) Start(stop <-chan struct{}) {
 	ctx := NewIstioContext(stop)
 	go c.cmInformer.Run(ctx.Done())
 
+	c.workers = NewQueue(10*time.Second, func(entry cacheEntry) {
```

Comment:
There should be a check that StatusMaxWorkers > 0 here, or perhaps NewQueue() should take an unsigned rather than an int.